fawltydeps/settings.py

```diff
@@ -215,9 +216,6 @@ def create(cls, cmdline_args: argparse.Namespace) -> "Settings":
         if {"verbose", "quiet"}.intersection(args_dict.keys()):
             ret["verbosity"] = args_dict.get("verbose", 0) - args_dict.get("quiet", 0)
 
-        if "custom_mapping" in args_dict.keys():
```

Comment:
The `cli_parser.py` file is not changed in this PR so I cannot comment on it, but shouldn't we change the argument name in that module from `custom-mapping` to `custom-mapping-path`?